packages/flutter_tools/lib/src/web/devfs_config.dart

```diff
@@ -0,0 +1,261 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+// import 'package:source_span/source_span.dart';
+import 'package:yaml/yaml.dart';
+
+import '../base/common.dart';
+import '../base/file_system.dart' as fs;
+import '../globals.dart' as globals;
```

Comment:
Nit: we're trying to reduce the reliance on `globals` for various reasons. For things like file system and logger instances, it's preferred that objects that need access to those resources take them in as parameters instead, particularly when we're adding new classes that don't already rely on `globals`.